ios/ReactNativeFingerprintScanner.m

```diff
@@ -26,6 +26,32 @@ @implementation ReactNativeFingerprintScanner
     }
 }
 
+RCT_EXPORT_METHOD(biometryType: (RCTResponseSenderBlock)callback)
```

Comment:
Can we make this promise compliant

```objective-c
       RCT_REMAP_METHOD(
                 biometryType,
                 findEventsWithResolver:(RCTPromiseResolveBlock)resolve
                 rejecter:(RCTPromiseRejectBlock)reject
      )
```

And do resolve, reject wherever necessary to make the api response same as `Android`.